File: solution/3_WordChains/3_WordChains/src/WordChain.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace wordchains
{
    public class WordChain
    {
        private List<String> possibleWords;

        public WordChain(List<String> possibleWords)
        {
            this.possibleWords = possibleWords;
        }

        /// <summary>
        /// Gebruik makend van de lijst van possible words, maakt wordchain een ketting van woorden die je van from naar to leiden met telkens
	    /// één letter verschil.
	    /// bvb cat -> dog : (cat, cot, cog, dog)
        /// </summary>
        /// <param name="from">from</param>
        /// <param name="to">to</param>
        /// <returns>wordchain from to</returns>
        public List<String> wordchainFromTo(String from, String to)
        {
            throw new NotImplementedException("implement me!");
        }
    }
}

File: workspace/3_WordChains/src/wordchains/WordChain.java
package wordchains;

import java.util.List;

public class WordChain {

	private List<String> possibleWords;

	public WordChain(List<String> possibleWords) {
		this.possibleWords = possibleWords;
	}

	/**
	 * Returns a chain of words going from aFrom to aTo where every link only changes one character
	 * Example cat -> dog : (cat, cot, cog, dog)
	 * @param aFrom from
	 * @param aTo to
	 * @return wordchain from to
	 */
	public List<String> wordchainFromTo(String aFrom, String aTo) {
		throw new UnsupportedOperationException("implement me!");
	}
	
	

}
